### ISSUE TITLE
Mobs

### ISSUE THREAD

**Issue by whichonespink44** [whichonespink44/Appalachia#10]
Appalachia biomes will generate mobs to help them achieve a more immersive aesthetic.

Passive mobs are preferred, but it's alright to have some mobs become hostile if provoked in some way (e.g. Black Bear attacks if you get too close its cub).

**Confirmed**
- [x] Black Bear + Baby Black Bear
  - [x] Adults drop Raw Bear
- [ ] White-tailed Deer + Baby White-tailed Deer
  - [ ] Male & female versions
  - [ ] Adults drop Raw Venison
  - [ ] Gets 'spooked' if you get too close, just like Ocelot
  - [ ] Should be faster than a player's normal sprint speed
- [x] Lightning Bug
  - [x] Maybe drops Glowstone Dust?
- [ ] Squirrel
  - [ ] Drops Raw Squirrel
  - [ ] Should be able to climb up and down trees

**Undecided**
- [ ] Beaver
- [ ] Snake (Garter Snake, Copperhead Snake, Timber Rattlesnake)
- [ ] Skunk
  - [ ] Sprays you if you get too close, giving you new potion effect 'Stench' which doesn't have any effects... you just stink. :)
- [ ] Bird
  - [ ] Mourning Dove
  - [ ] Warbler
  - [ ] Cardinal
  - [ ] Blue-Jay
  - [ ] Robin
  - [ ] Woodpecker
- [ ] Fish
- [ ] Cayote

**Comment by whichonespink44** [comment]
With regards to lightning bugs... if there's a way to achieve the 'lightning bug effect' without having to generate a ton of mobs, we should probably go down that route, but I'm not sure if that's even possible.

**Comment by Sunconure11** [comment]
What about some kind of biome-exclusive particle? I think twilight forest does something like that.

**Comment by lightningo7** [comment]
I definitely agree with the idea you have for black bears having an aggro range based on its cubs. For deers, they should probably be pretty fast (faster than a sprint). Would be cool to see squirrels climbing trees and the sort. If you decide on doing snake, you may want to point out the type of snake so as to make how to handle it's aggressiveness a bit more simple. Would be cool to see Birds and Fish (Birds just because you see them everywhere in real life if we're making it realistic may as well do them too, right? Fish because what do the black bears eat?)

**Comment by whichonespink44** [comment]
+1 for 'faster-than-you' deer
+1 for tree-climbing squirrels - that definitely has to be a thing! :)

I'm not too bothered about having bird mobs, but we definitely need bird songs. (See #9 for details). Although, having said that, it _would_ be very cool to see some birds flying around whilst exploring.

Fish would also be cool, and I'd love to approach them the same way Tmtravlr did them in Just a Few Fish for 1.7.10/1.8.9, which was to basically have mobs for each of the vanilla fish types, but I'm conscious that we've already got quite a few mobs on the table, and if we had to draw the line somewhere, I could easily live without fish.

Updated OP with more notes.
